### PR TITLE
fix: removeEdge not effects on graph in golang

### PR DESCRIPTION
**Pull request by shanghai-Jerry**
If this PR is related to coding or code translation, please fill out the checklist and paste the console outputs to the PR.

- [x] I've tested the code and ensured the outputs are the same as the outputs of reference codes.
- [x] I've checked the codes (formatting, comments, indentation, file header, etc) carefully.
- [x] The code does not rely on a particular environment or IDE and can be executed on a standard system (Win, macOS, Ubuntu).

fix: #502 ， got outputs：
=== RUN   TestGraphAdjList
初始化后，图为:
邻接表 =
                5: 1 2 4
                2: 3 5 4
                4: 2 5
                1: 3 5
                3: 1 2

添加边 1-2 后，图为
邻接表 =
                1: 3 5 2
                3: 1 2
                5: 1 2 4
                2: 3 5 4 1
                4: 2 5

删除边 1-3 后，图为
邻接表 =
                1: 5 2
                3: 2
                5: 1 2 4
                2: 3 5 4 1
                4: 2 5

添加顶点 6 后，图为
邻接表 =
                4: 2 5
                6:
                1: 5 2
                3: 2
                5: 1 2 4
                2: 3 5 4 1

删除顶点 3 后，图为
邻接表 =
                1: 5 2
                5: 1 2 4
                2: 5 4 1
                4: 2 5
                6:
--- PASS: TestGraphAdjList (0.00s)

### CODE REVIEW

File: codes/go/chapter_graph/graph_adjacency_list.go [hunk 1]
```diff
@@ -98,3 +100,32 @@ func (g *graphAdjList) print() {
 		builder.Reset()
 	}
 }
+
+// graph 比较， 检查是否是同一个图
+func isSameGraph(g1, g2 *graphAdjList) bool {
```
Comment (krahets) [inline codes/go/chapter_graph/graph_adjacency_list.go:105]: Please remove this function. We should keep the Go code consistent with others.

File: codes/go/go.mod [hunk 2]
```diff
@@ -1,3 +1,11 @@
 module github.com/krahets/hello-algo
 
 go 1.19
+
+require github.com/stretchr/testify v1.8.2
```
Comment (krahets) [inline codes/go/go.mod:5]: Is the added package necessary? Remove them if not.

**Review by krahets — COMMENTED** [review]
Thanks for the great fix. Please address the comments.

File: codes/go/chapter_graph/graph_adjacency_list.go (hunk 1 above)
Comment (shanghai-Jerry) [reply codes/go/chapter_graph/graph_adjacency_list.go:105]: 这个主要是为了方便检查图是否一致，通过print的形式查看比较不直观

Comment (krahets) [reply codes/go/chapter_graph/graph_adjacency_list.go]: 确实。不过添加代码需要开启多个种语言的同步工作，暂时觉得没有必要添加这一段

**Comment by krahets** [comment]
Please limit the code modification in the PR to the topic `fix: removeEdge not effects on graph` and I'm about to merge it.

**Comment by shanghai-Jerry** [comment]
>  ok，i am ready to remove unnecessary package in this PR

**Comment by shanghai-Jerry** [comment]
@krahets  Done now the modification is simple and looks good. PTAL

**Pull request merged**